package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.7.20-prerelease.1",
+  "version": "1.8.0",
```

Comment:
since the admin auth is here, i'd expect this to be 1.8.1-prerelease.1

Reply:
Sure